hack/configure_local_webhook.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -ex
+
+TMPDIR=${TMPDIR:-"/tmp/k8s-webhook-server/serving-certs"}
+
+mkdir -p ${TMPDIR}
+rm -rf ${TMPDIR}/* || true
+
+openssl req -newkey rsa:2048 -days 3650 -nodes -x509 \
```

Comment:
would it make sense to use cert-manager for development purposes only, and just issue ACME certs with letsencrypt?

I know its inclusion under the product is under question, just I believe we still could use it for testing purposes, like this. And avoid all the manual certs management